guides/creating-a-custom-stepper-using-the-cdk-stepper.md

```diff
@@ -0,0 +1,187 @@
+# Creating a custom stepper using the CDK stepper
+
+Using the [CDK stepper](https://material.angular.io/cdk/stepper/overview) it is possible to build a custom stepper which you can completely style yourself without any specific Material Design styling.
+
+In this guide, we'll learn how we can build our own custom stepper using the CDK stepper. Here is what we'll build by the end of this guide:
+
+<!-- example(cdk-custom-stepper-without-form) -->
+
+## Add CdkStepperModule to your project 
+
+After adding the Angular CDK to your Angular project, the next step is to add the `CdkStepperModule` to your Angular module:
+
+```typescript
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { CdkStepperModule } from '@angular/cdk/stepper'; // this is the relevant import
+
+import { AppComponent } from './app.component';
+
+@NgModule({
+  imports: [BrowserModule, CdkStepperModule], // add the module to your imports
+  declarations: [AppComponent],
+  bootstrap: [AppComponent],
+})
+export class AppModule {}
+```
```

Comment:
Generally we assume a baseline level of familiarity with Angular for our docs and try to avoid code examples like "This is how you add an NgModule"

Reply:
Done